src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  return (
+    <div>
+      <div className="container">
+        <Button btnLabel="AC" />
+        <Button btnLabel="+/-" />
+        <Button btnLabel="%" />
+        <Button btnLabel="/" color="orange" />
+      </div>
+      <div className="container">
+        <Button btnLabel="7" />
+        <Button btnLabel="8" />
+        <Button btnLabel="9" />
+        <Button btnLabel="X" color="orange" />
+      </div>
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.